helper.psm1

```diff
@@ -152,6 +152,19 @@ Function unlockDirectory
 # Configure Oskar
 ################################################################################
 
+Function configureCache
+{
+    If($env:CLCACHE_CL)
+    {
+	    proc -process "$(Split-Path $env:CLCACHE_CL)\cl.exe" -argument "-M 107374182400" -logfile $false -priority "Normal"
+	    proc -process "$(Split-Path $env:CLCACHE_CL)\cl.exe" -argument "-s" -logfile $false -priority "Normal"
```

Comment:
Just installed clcache-4.1.0 via nuget and see that binary name is `clcache.exe`. Is that correct?

Reply:
I install clcache 4.2 via zip file and rename it to work properly